static_src/components/stat.jsx

```diff
@@ -0,0 +1,52 @@
+
+import React from 'react';
+
+import style from 'cloudgov-style/css/cloudgov-style.css';
+
+import createStyler from '../util/create_styler';
+
+const STATES = [
+  'error',
+  'warning',
+  'info',
+  'success',
+  'none'
+];
+
+const propTypes = {
+  primaryStat: React.PropTypes.string.isRequired,
+  statState: React.PropTypes.oneOf(STATES),
+  secondaryInfo: React.PropTypes.element
```

Comment:
Does this need the customizability to be an element? Or can it just be a string?

I noticed in the UI, it's color should change along with that statState, but it should only change when the stat is red or is being edited. Should that more complex logic be held in the ResourceUsage component or here? That's kinda the decision to be an element or text.